Form-Controls/index.html

```diff
@@ -13,15 +13,66 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form action="result.html" method="get">
+        <div class="form-element">
+          <label for="name">Name</label>
+          <input type="text" name="name" id="name" placeholder="username" required>
+        </div>
+        <div class="form-element">
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" 
+            placeholder="example@domain.com" required>
+        </div>
+        <fieldset>
+          <legend>Please select color:</legend>
+            <div class="checks">
+              <input type="radio" id="red" name="color"/>
+              <label for="red">Red</label>
+
+              <input type="radio" id="yellow" name="color"/>
+              <label for="yellow">Yellow</label>
+              
+              <input type="radio" id="Blue" name="color"/>
```

Comment:
```suggestion
              <input type="radio" id="blue" name="color"/>
```
does it matter if ids are capitalised? Have a think